modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/vulpkanin.dm

```diff
@@ -7,7 +7,8 @@
 		LIPS,
 		HAS_FLESH,
 		HAS_BONE,
-		HAIR
+		HAIR,
+		FACEHAIR
```

Comment:
Only thing that needs changing before we can merge this.
```suggestion
		FACEHAIR,
```

Reply:
Here I was, thinking a single word-change PR would be a breeze... Thanks!